doc/development/sprint_review_presentation.md

```diff
@@ -0,0 +1,80 @@
+# Sprint Review Presentation
+
+Creating a good presentation for a **Sprint Review** in PAF is important for communicating the team's achievements, identifying challenges, and planning for future work.
```

Comment:
Should we enforce the template for the wiki files? If so a "**Summary:**" should be added at the beginning of this line.